src/components/fields/SchemaField.js

```diff
@@ -87,13 +87,14 @@ function DefaultTemplate(props) {
     hidden,
     required,
     displayLabel,
+    formContext,
```

Comment:
Please update field template docs to reflect this prop is now passed to custom template components as well.